.github/workflows/UnitTests.yml

```diff
@@ -49,8 +49,10 @@ jobs:
         pytype --jobs auto --disable import-error MaxText/
     - name: Analysing the code with pylint
       run: |
-        pylint MaxText/
-
+        pylint MaxText/ pedagogical_examples/ \
+         --fail-under=9 \
```

Comment:
Instead of using command line options here, please move the config variables inside pylintrc file. 
recursive can go under [MASTER] and check-quote-consistency can go under the [FORMAT] section

Reply:
Sounds good, done.